shared/src/test/scala/coop/rchain/shared/SyncInstancesSpec.scala

```diff
@@ -0,0 +1,75 @@
+package coop.rchain.shared
+
+import cats.implicits._
+import cats.data.EitherT
+import cats.effect.{ExitCase, Sync}
+import cats.effect.concurrent.Ref
+import coop.rchain.catscontrib.SyncInstances.syncEffect
+import monix.eval.Task
+import monix.execution.Scheduler.Implicits.global
+import org.scalatest.{FunSpec, Matchers}
+
+import scala.concurrent.duration._
+
+class SyncInstancesSpec extends FunSpec with Matchers {
```

Comment:
I guess the above is no longer valid. Or maybe it is - let's have that test anyway so that whatever Effect we'll have underneath, the laws are checked for it and hold.